projects/mysadna2/src/app/app.component.scss

```diff
@@ -0,0 +1,16 @@
+.app-navigation {
+    float: right;
+    >a {
+        margin: 5px;
+        width: 83px;
+        height: 24px;
+        font-family: OpenSans;
+        font-size: 18px;
+        font-weight: bold;
+        font-style: normal;
+        font-stretch: normal;
+        line-height: normal;
+        letter-spacing: normal;
+        color: #000000;
+    }
+}
```

Comment:
end without new line

Reply:
Already fixed in more recent commit yet to have been uploaded.